### PR TITLE
Python bindings + exp/log functions

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -85,12 +86,12 @@ target_include_directories(qserl
...
   )
 
-target_compile_features(qserl PRIVATE cxx_std_11)
+#target_compile_features(qserl PRIVATE cxx_std_11)
 target_compile_options(qserl PRIVATE -Wall -Wextra)
 
 target_link_libraries(qserl
   PUBLIC
-    Boost::boost
+  ${Boost_LIBRARIES}
```
Comment (olivier-roussel) [inline CMakeLists.txt]: I would prefer keeping the use of targets:
`Boost::boost
Boost::python`

File: CMakeLists.txt [hunk 2]
```diff
@@ -85,12 +86,12 @@ target_include_directories(qserl
     ${CMAKE_CURRENT_SOURCE_DIR}/src
   )
 
-target_compile_features(qserl PRIVATE cxx_std_11)
+#target_compile_features(qserl PRIVATE cxx_std_11)
```
Comment (olivier-roussel) [inline CMakeLists.txt]: Maybe we could keep the c++11 per-target feature for cmake versions that are compatible, like this:
`if(${CMAKE_MAJOR_VERSION} EQUAL 3 AND ${CMAKE_MINOR_VERSION} GREATER_EQUAL 8)`
`  target_compile_features(qserl PRIVATE cxx_std_11)`
`else()`
`  set(CMAKE_CXX_STANDARD 11)`
`endif()`
Sounds ok for you ?

File: CMakeLists.txt (hunk 1 above)
Comment (jmirabel) [reply CMakeLists.txt]: Done

File: CMakeLists.txt (hunk 2 above)
Comment (jmirabel) [reply CMakeLists.txt]: Done.

**Comment by jmirabel** [comment]
I updated the licenses. I also added a class to do inverse kinematics which I can remove if you do not want it.

**Comment by olivier-roussel** [comment]
Great, thanks.

**Pull request merged**